
PLAY [reco_release] **************************************************************************************************************************************************************

TASK [Gathering Facts] ***********************************************************************************************************************************************************
ok: [10.11.5.115]
ok: [10.11.5.113]
ok: [10.11.5.111]
ok: [10.11.5.112]
ok: [10.11.5.117]
ok: [10.11.5.116]
ok: [10.11.5.118]
ok: [10.11.5.119]
ok: [10.11.5.122]
ok: [10.11.5.120]
ok: [10.11.5.121]
ok: [10.11.5.123]
ok: [10.11.5.124]
ok: [10.11.5.125]
ok: [10.11.5.126]
ok: [10.11.5.127]
ok: [10.11.5.114]
ok: [10.11.5.128]
ok: [10.11.5.130]
ok: [10.11.5.131]
ok: [10.11.5.132]
ok: [10.11.5.129]
ok: [10.11.5.134]
ok: [10.11.5.133]

TASK [reco_release : pull reco images k] *****************************************************************************************************************************************
fatal: [10.11.5.112]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:19.915415", "end": "2018-11-13 11:45:01.267344", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:44:41.351929", "stderr": "Error response from daemon: Get https://registry-1.docker.io/v2/library/k/manifests/latest: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fk%3Apull&service=registry.docker.io: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)", "stderr_lines": ["Error response from daemon: Get https://registry-1.docker.io/v2/library/k/manifests/latest: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fk%3Apull&service=registry.docker.io: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.115]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:20.008902", "end": "2018-11-13 11:45:01.318397", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:44:41.309495", "stderr": "Error response from daemon: Get https://registry-1.docker.io/v2/library/k/manifests/latest: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fk%3Apull&service=registry.docker.io: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)", "stderr_lines": ["Error response from daemon: Get https://registry-1.docker.io/v2/library/k/manifests/latest: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fk%3Apull&service=registry.docker.io: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.113]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:20.012802", "end": "2018-11-13 11:45:01.336531", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:44:41.323729", "stderr": "Error response from daemon: Get https://registry-1.docker.io/v2/library/k/manifests/latest: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fk%3Apull&service=registry.docker.io: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)", "stderr_lines": ["Error response from daemon: Get https://registry-1.docker.io/v2/library/k/manifests/latest: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fk%3Apull&service=registry.docker.io: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.111]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:20.028188", "end": "2018-11-13 11:45:01.354795", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:44:41.326607", "stderr": "Error response from daemon: Get https://registry-1.docker.io/v2/library/k/manifests/latest: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fk%3Apull&service=registry.docker.io: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)", "stderr_lines": ["Error response from daemon: Get https://registry-1.docker.io/v2/library/k/manifests/latest: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fk%3Apull&service=registry.docker.io: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.114]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:20.049583", "end": "2018-11-13 11:45:01.469840", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:44:41.420257", "stderr": "Error response from daemon: Get https://registry-1.docker.io/v2/library/k/manifests/latest: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fk%3Apull&service=registry.docker.io: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)", "stderr_lines": ["Error response from daemon: Get https://registry-1.docker.io/v2/library/k/manifests/latest: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fk%3Apull&service=registry.docker.io: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.120]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:15.928518", "end": "2018-11-13 11:45:17.659314", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:01.730796", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.119]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:16.361568", "end": "2018-11-13 11:45:18.092473", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:01.730905", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.116]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:17.185730", "end": "2018-11-13 11:45:18.768651", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:01.582921", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.118]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:18.014251", "end": "2018-11-13 11:45:19.695906", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:01.681655", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.117]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:18.859845", "end": "2018-11-13 11:45:20.498453", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:01.638608", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.121]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:16.813420", "end": "2018-11-13 11:45:34.853330", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:18.039910", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.122]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:16.513378", "end": "2018-11-13 11:45:34.908556", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:18.395178", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.123]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:18.427910", "end": "2018-11-13 11:45:37.607109", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:19.179199", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.124]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:18.129198", "end": "2018-11-13 11:45:38.262427", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:20.133229", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.125]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:18.376848", "end": "2018-11-13 11:45:39.269780", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:20.892932", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.127]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:18.546427", "end": "2018-11-13 11:45:53.735592", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:35.189165", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.126]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:18.639089", "end": "2018-11-13 11:45:53.897535", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:35.258446", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.128]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:18.526842", "end": "2018-11-13 11:45:56.500318", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:37.973476", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.130]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:16.129589", "end": "2018-11-13 11:45:57.027262", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:40.897673", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.129]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:18.731251", "end": "2018-11-13 11:45:57.346673", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:38.615422", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.132]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:12.891687", "end": "2018-11-13 11:46:08.484535", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:55.592848", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.131]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:14.136713", "end": "2018-11-13 11:46:08.705196", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:54.568483", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.133]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:12.257648", "end": "2018-11-13 11:46:09.493473", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:57.235825", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.134]: FAILED! => {"changed": true, "cmd": "docker pull k", "delta": "0:00:13.499467", "end": "2018-11-13 11:46:11.237729", "msg": "non-zero return code", "rc": 1, "start": "2018-11-13 11:45:57.738262", "stderr": "Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for k, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
	to retry, use: --limit @/home/op/cheng/autohdmap_deploy/reco_release.retry

PLAY RECAP ***********************************************************************************************************************************************************************
10.11.5.111                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.112                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.113                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.114                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.115                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.116                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.117                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.118                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.119                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.120                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.121                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.122                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.123                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.124                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.125                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.126                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.127                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.128                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.129                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.130                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.131                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.132                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.133                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.134                : ok=1    changed=0    unreachable=0    failed=1   

[Finish] Failed to deploy [k]